src/pages/EditCardPage/EditCardPageHeader.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react'
+import Button from '../../components/UI/Button/Button'
+import classes from './EditCardPageHeader.module.css'
+import { useHistory } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+import DeckForm from './DeckForm'
+import { useDispatch } from 'react-redux'
+import { onSaveTitle } from '../../features/deckTitle/deckSlice'
+
+const EditCardPageHeader = (props) => {
+    const history = useHistory()
+    const dispatch = useDispatch()
+
+    const [showEditForm, setShowEditForm] = useState()
+    const title = useSelector((state) => state.deck.title)
+
+    const onClickTitleEditHandler = () => {
+        setShowEditForm(true)
+    }
+
+    const clickAddCardButton = () => {
```

Comment:
Similar to the previous PR with the edit button on created cards form, if the user clicks Add Card and then cancel maybe we should have it route back to /edit-card